Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -6168,7 +6173,11 @@ describe(
       it("renders implicit tileset with multiple contents (legacy)", function () {
```

Comment:
looks like the test
`renders implicit tileset with multiple contents (legacy with 'content')` also has the same issue, let's disable that one too.